pkgs/by-name/cr/crates-tui/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  nix-update-script,
+}:
+let
+  version = "0.1.20";
+in
+rustPlatform.buildRustPackage {
+  pname = "crates-tui";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "ratatui";
+    repo = "crates-tui";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-K3ttXoSS4GZyHnqS95op8kmbAi4/KjKa0P6Nzqqpjyw=";
+  };
+
+  cargoHash = "sha256-ztLBM6CR2WMKR9cfBY95BvSaD05C+AEa6C/nOdDxqf0=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "TUI for exploring crates.io using Ratatui";
+    homepage = "https://github.com/ratatui/crates-tui";
+    license = with lib.licenses; [ mit ];
```

Comment:
```suggestion
    license = lib.licenses.mit;
```

Reply:
I prefer it this way since it allows for more licenses to be added in the future (besides, `meta.license` is a misleading name anyway)